packages/components/src/List/DisplayTable/DisplayTable.component.js

```diff
@@ -176,7 +178,7 @@ export function ColumnHeader({ index, column, sort }) {
 	return (
 		<th key={index}>
 			{header}
-			<div aria-hidden="true">{column.label}</div>
+			{ !column.hideHeader ? (<div aria-hidden="true">{column.label}</div>) : null }
```

Comment:
You can write `{ column.hideHeader || (<div aria-hidden="true">{column.label}</div>)}`